src/node/interop/interop_client.js

```diff
@@ -318,6 +318,58 @@ function authTest(expected_user, scope, client, done) {
   });
 }
 
+function oauth2Test(expected_user, scope, per_rpc, client, done) {
+  (new GoogleAuth()).getApplicationDefault(function(err, credential) {
+    assert.ifError(err);
+    var arg = {
+      response_type: 'COMPRESSABLE',
```

Comment:
Oh, I didn't even notice. I just copied from the service account test. I'll delete that stuff to conform to the spec.